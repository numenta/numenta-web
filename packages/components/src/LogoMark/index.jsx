// Numenta Web Platform and Sites source code
// MIT License (see LICENSE.txt)
// Copyright © 2005—2017 Numenta <http://numenta.com>

import React from 'react'

import Image from '../Image'

import LogoFile from './images/mark.svg'
import styles from './index.css'


/**
 * Numenta Logo Mark - React view component.
 */
const LogoMark = () => (
  <Image
    alt="Numenta Logo Mark"
    className={styles.mark}
    shadow={false}
    src={LogoFile}
    title="Numenta"
  />
)

export default LogoMark
