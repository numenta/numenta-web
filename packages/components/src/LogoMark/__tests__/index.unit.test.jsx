import React from 'react'
import renderer from 'react-test-renderer'

import LogoMark from '../../LogoMark'


describe('LogoMark React component', () => {

  it('Renders correctly', () => {
    const component = renderer.create(
      <LogoMark />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

})
