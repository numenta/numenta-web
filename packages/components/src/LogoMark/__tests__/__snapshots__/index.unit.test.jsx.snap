exports[`LogoMark React component Renders correctly 1`] = `
<img
  alt="Numenta Logo Mark"
  className="image"
  onClick={undefined}
  src="filename.ext"
  title="Numenta" />
`;
